fix: adicionado o link de aceitar e rejeitar solicitacao

MateuxLucax opened the pull request.

MateuxLucax committed:
fix: adicionado o link de aceitar e rejeitar solicitacao

MateuxLucax committed:
tests: fix link no teste NovaInclusaoMail

ghost commented:
![Passed](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/QualityGateBadge/passed.svg?sanitize=true)

# Analysis Details
## 0 Issues
- ![Bug](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/bug.svg?sanitize=true) 0 Bugs
- ![Vulnerability](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/vulnerability.svg?sanitize=true) 0 Vulnerabilities
- ![Code Smell](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/code_smell.svg?sanitize=true) 0 Code Smells

## Coverage and Duplications
- ![Coverage](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/CoverageChart/NoCoverageInfo.svg?sanitize=true) No coverage information (100.00% Estimated after merge)
- ![Duplications](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/Duplications/NoDuplicationInfo.svg?sanitize=true) No duplication information (0.00% Estimated after merge)

**Project ID:** MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7

[View in SonarQube](https://sonarqube.mateux.dev/dashboard?id=MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7&pullRequest=45)

MateuxLucax committed:
Merge branch 'main' into fix-add-link-aceite-rejeita-solic-email

ghost commented:
![Passed](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/QualityGateBadge/passed.svg?sanitize=true)

# Analysis Details
## 0 Issues
- ![Bug](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/bug.svg?sanitize=true) 0 Bugs
- ![Vulnerability](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/vulnerability.svg?sanitize=true) 0 Vulnerabilities
- ![Code Smell](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/code_smell.svg?sanitize=true) 0 Code Smells

## Coverage and Duplications
- ![Coverage](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/CoverageChart/NoCoverageInfo.svg?sanitize=true) No coverage information (100.00% Estimated after merge)
- ![Duplications](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/Duplications/NoDuplicationInfo.svg?sanitize=true) No duplication information (0.00% Estimated after merge)

**Project ID:** MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7

[View in SonarQube](https://sonarqube.mateux.dev/dashboard?id=MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7&pullRequest=45)

luc527 reviewed (changes requested):
Os links só funcionam se o técnico estiver logado, e a ideia dos links dos e-mails é justamente funcionar sem ele estar necesariamente logado

![image](https://github.com/MateuxDotDev/badminton-manager/assets/48414967/ee247cb3-cc1e-41fe-bca3-94f3119e2696)

luc527 commented:
> Os links só funcionam se o técnico estiver logado, e a ideia dos links dos e-mails é justamente funcionar sem ele estar necesariamente logado
> 
> ![image](https://user-images.githubusercontent.com/48414967/248521043-ee247cb3-cc1e-41fe-bca3-94f3119e2696.png)

Além disso, teria que cuidar para as ações de aceitar e rejeitar funcionem com o token também (ao clicar nos botões para confirmar).

MateuxLucax committed:
Merge branch 'main' into fix-add-link-aceite-rejeita-solic-email

ghost commented:
![Passed](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/QualityGateBadge/passed.svg?sanitize=true)

# Analysis Details
## 0 Issues
- ![Bug](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/bug.svg?sanitize=true) 0 Bugs
- ![Vulnerability](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/vulnerability.svg?sanitize=true) 0 Vulnerabilities
- ![Code Smell](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/code_smell.svg?sanitize=true) 0 Code Smells

## Coverage and Duplications
- ![Coverage](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/CoverageChart/NoCoverageInfo.svg?sanitize=true) No coverage information (100.00% Estimated after merge)
- ![Duplications](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/Duplications/NoDuplicationInfo.svg?sanitize=true) No duplication information (0.00% Estimated after merge)

**Project ID:** MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7

[View in SonarQube](https://sonarqube.mateux.dev/dashboard?id=MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7&pullRequest=45)

MateuxLucax committed:
fix: adicionado login via token

ghost commented:
![Passed](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/QualityGateBadge/passed.svg?sanitize=true)

# Analysis Details
## 0 Issues
- ![Bug](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/bug.svg?sanitize=true) 0 Bugs
- ![Vulnerability](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/vulnerability.svg?sanitize=true) 0 Vulnerabilities
- ![Code Smell](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/code_smell.svg?sanitize=true) 0 Code Smells

## Coverage and Duplications
- ![Coverage](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/CoverageChart/0.svg?sanitize=true) 0.00% Coverage (99.60% Estimated after merge)
- ![Duplications](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/Duplications/3.svg?sanitize=true) 0.00% Duplicated Code (0.00% Estimated after merge)

**Project ID:** MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7

[View in SonarQube](https://sonarqube.mateux.dev/dashboard?id=MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7&pullRequest=45)

luc527 committed:
correção ao aceitar/rejeitar solicitação por e-mail

O erro que acontecia era o seguinte:
Quando o técnico aceita ou rejeita pelo e-mail, após confirmar a ação, era feito um reload (linha 329 da /tecnico/solicitacoes/index.php). Nesse reload a URL não muda e fica com o &solicitacao=123&acao=aceitar&etc. O problema é que ele tenta realizar a ação novamente sendo que a solicitação acabou de ser deletada, aí mostrava o erro de "Solicitação não encontrada".

ghost commented:
![Passed](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/QualityGateBadge/passed.svg?sanitize=true)

# Analysis Details
## 0 Issues
- ![Bug](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/bug.svg?sanitize=true) 0 Bugs
- ![Vulnerability](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/vulnerability.svg?sanitize=true) 0 Vulnerabilities
- ![Code Smell](https://sonarqube.mateux.dev/static/communityBranchPlugin/common/code_smell.svg?sanitize=true) 0 Code Smells

## Coverage and Duplications
- ![Coverage](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/CoverageChart/0.svg?sanitize=true) 0.00% Coverage (99.60% Estimated after merge)
- ![Duplications](https://sonarqube.mateux.dev/static/communityBranchPlugin/checks/Duplications/3.svg?sanitize=true) 0.00% Duplicated Code (0.00% Estimated after merge)

**Project ID:** MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7

[View in SonarQube](https://sonarqube.mateux.dev/dashboard?id=MateuxDotDev_badminton-manager_AYfNu-knGxuymmoUfIu7&pullRequest=45)
